css/_uswds-theme-custom-styles.scss

```diff
@@ -1568,47 +1568,55 @@ iframe {
   margin-bottom: units(4);
 }
 
-.post-excerpt {
-  margin-top: units(5);
+.post-preview {
+  @include u-border-top("1px", "gray-30");
+  @include u-margin-y(3);
+  @include u-measure(6);
 
-  .post-title {
-    margin-top: 0;
-    margin-bottom: 0;
+  &__title,
+  &__content,
+  &__header,
+  .post-category {
+    @include u-margin-y(2);
+  }
+
+  &__header {
+    @include u-margin-top(3);
+  }
+
+  &__title {
+    @include typeset("lang", "lg", 3);
+    @include u-text("bold");
 
     a {
       color: inherit;
-      text-decoration: none;
 
       &:hover {
         color: color($site-primary);
-        text-decoration: underline;
       }
     }
   }
 
-  .post-category {
-    margin-top: 0;
-  }
-
-  .post-category + .post-title {
-    margin-top: 0;
-  }
-
-  .post-content p {
-    margin-bottom: 0;
-    margin-top: units(0.5);
+  &__content p {
+    @include u-font("lang", "xs");
   }
-} // .post-excerpt
+} // .post-preview
 
-.post-date {
+.post-date.site-subheading {
```

Comment:
**Question:** Are these only targetting this meta section underneath the title?

![image](https://github.com/user-attachments/assets/16bd3e99-03a0-47bc-9ff5-88ec37e121c3)

If so, would it make sense to unify them under one class name like `post-meta` since it's being included via `post-meta` include?

Reply:
Replaced `.post-date` with `.post-meta` in [55d9f08](https://github.com/uswds/uswds-site/pull/2979/commits/55d9f08c03ea8ce1055b6dc495a9edeebf055c3e)